Automattic/wp-calypso
# Add Email/Titan mailbox provisioning

#### Changes proposed in this Pull Request

This PR is the first step towards letting users fully specify their Email/Titan mailboxes in Calypso before checkout. This specific PR only includes the following changes:

* Create a new `titan/provision-mailboxes` feature flag to guard the new feature.
* Refactor some notices displayed in the existing quantity selection page into separate components in preparation for re-use in a new page for specifying mailbox details.

Later PRs will pull in the actual components and logic.

#### Testing instructions

For this PR, we need to test the following cases:
* Verify that you see an appropriate warning if you have unused mailboxes, and can click through to finish the setup.
* Verify that you see an appropriate warning if you have any email forwards set up for the domain.
* Verify that you see the correct notice about pricing if you're buying mailboxes for a domain (especially one where the subscription started on a different day).
